Source/IGListKit/IGListAdapterUpdaterCompatible.h

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Please remove this file, it's not in master anymore

Reply:
done